src/components/AssetTableHeader.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
```

Comment:
anything in here you'd like to test?

Reply:
Maybe getNextQuery? The rest of it is essentially wiring.